### PR TITLE
Alert/create a separate namespace

### PR DESCRIPTION
**Pull request by ghost-not-in-the-shell**
Explain your changes:
This PR fixes `testnet-alert-rules.tpl.yml`. There's a limit for the number of alerts in a namespace. To make the number of alerts in `Berkeley alerts` namespace below this limit, we create a new namespace "Berkeley archive node alerts"

Explain how you tested your changes:
*

Checklist:

- [ ] Dependency versions are unchanged
  - Notify Velocity team if dependencies must change in CI
- [ ] Modified the current draft of release notes with details on what is completed or incomplete within this project
- [ ] Document code purpose, how to use it
  - Mention expected invariants, implicit constraints
- [ ] Tests were added for the new behavior
  - Document test purpose, significance of failures
  - Test names should reflect their purpose
- [ ] All tests pass (CI will check this if you didn't)
- [ ] Serialized types are in stable-versioned modules
- [ ] Does this close issues? List them

* Closes #0000

### CODE REVIEW

File: automation/terraform/modules/testnet-alerts/templates/testnet-alert-rules.yml.tpl [hunk 1]
```diff
@@ -735,26 +713,49 @@ groups:
...
       testnet: "{{ $labels.testnet }}"
       severity: warning
     annotations:
-      summary: "{{ $labels.testnet }} rate of archival of network blocks in Postgres DB is lower than expected"
-      description: "The rate of {{ $value }} new blocks observed by archive postgres instances is low on network {{ $labels.testnet }}."
+      summary: "At least one of the nodes on {{ $labels.testnet }} restarted"
+      description: "{{ $value }} nodes on {{ $labels.testnet }} restarted"
+      runbook: "https://www.notion.so/minaprotocol/NodeRestarted-99a1cf710ff14aa6930a9f12ad5813a5"
+
+- Berkeley archive node alerts
```
Comment (dkijania) [inline automation/terraform/modules/testnet-alerts/templates/testnet-alert-rules.yml.tpl:726]: shouldn't we prefix this with `name:`
like:
`- name: Critical Alerts`

Comment (ghost-not-in-the-shell) [reply automation/terraform/modules/testnet-alerts/templates/testnet-alert-rules.yml.tpl:726]: Nice catch.

**Comment by ghost-not-in-the-shell** [comment]
!ci-build-me

**Comment by ghost-not-in-the-shell** [comment]
!ci-build-me

**Review by dkijania — APPROVED** [review]

**Comment by ghost-not-in-the-shell** [comment]
!ci-build-me

**Comment by mrmr1993** [comment]
!approved-for-mainnet

**Comment by mrmr1993** [comment]
!approved-for-mainnet

**Comment by ghost-not-in-the-shell** [comment]
!ci-build-me

**Comment by ghost-not-in-the-shell** [comment]
!ci-build-me

**Comment by ghost-not-in-the-shell** [comment]
!ci-build-me

**Comment by ghost-not-in-the-shell** [comment]
!ci-build-me

**Comment by deepthiskumar** [comment]
!ci-build-me

**Comment by deepthiskumar** [comment]
!approved-for-mainnet

**Pull request merged**